### DISCUSSION TITLE
How to handle expenses with multiple source accounts

### DISCUSSION THREAD

**Discussion by gobo7793** [firefly-iii/firefly-iii#9064] [Firefly III]
I use firefly since the beginning of the year and it's a nice tool. But sometimes when I pay with Paypal and my paypal balance isn't enough to pay everything I've set to get the rest from my credit card. I know there is the feature with splitted transactions, which would be nice to use this for such cases. But it's not possible to use difference source accounts for splitted expense transactions, but I can set different target accounts.

What's the sense of this? I would expect the opposite that the target account needs to be the same. Do I really need to add the same transaction twice with different source accounts?

**Comment by digitlength** [comment]
For me this is legitimately two different transactions. 

1. From credit card account to PayPal account
2. From PayPal account to vendor
